### DISCUSSION TITLE
Google Analytics

### DISCUSSION THREAD

**Discussion by benjamincarmel** [requarks/wiki#4173] [Error / Bug Report] [answered]
Google Analytics has moved to a new code snippet and deprecated UA analytics codes. Could support for the new code snippet which starts with G instead of UA be added?

**Comment by benry1** [comment]
I'm having the same issue. We can't generate a new UA code for analytics, only the new G- code

**Comment by NGPixel** [comment, marked as answer]
You can already use the new `G-` codes...
